containers/general/tool.go

```diff
@@ -62,6 +63,7 @@ type GeneralConfigs struct {
 	Okta          usp_okta.OktaConfig                `json:"okta" yaml:"okta"`
 	Office365     usp_o365.Office365Config           `json:"office365" yaml:"office365"`
 	Wel           usp_wel.WELConfig                  `json:"wel" yaml:"wel"`
+	Mac	      usp_mac.MacConfig			 `json:"mac" yaml:"mac"`
```

Comment:
Could we rename it to something like "mac_unified_log", just to avoid future collision with other stuff on a Mac.